syncapi/consumers/eduserver_sendtodevice.go

```diff
@@ -107,7 +107,7 @@ func (s *OutputSendToDeviceEventConsumer) onMessage(msg *sarama.ConsumerMessage)
 	s.notifier.OnNewSendToDevice(
 		output.UserID,
 		[]string{output.DeviceID},
-		types.NewStreamToken(0, streamPos, nil),
+		types.NewStreamToken(0, 0, 0, streamPos, nil),
```

Comment:
I think it would be better if NewStreamingToken accepted a struct at this point so we can just omit 0 values and get more intuitive labels on what stream we're using rather than relying on the position.

Reply:
If that's the case, we had might as well just remove `NewStreamToken` and `NewTopologyToken` altogether and instantiate the structs directly :-)

Reply:
Maybe we should then?

Reply:
I'm happy to go through and do that tomorrow!

Reply:
Did this now, as it didn't take long.